doc/vaulted-shell.1.md

```diff
@@ -22,8 +22,8 @@ OPTIONS
 -------
 
 `--assume` *arn*
-  Specifies the full ARN of the role to assume. See **ASSUMING A ROLE** below
-  for details on how Vaulted assumes roles.
+  Specifies the full ARN or the shortname of the role to assume. Se
```

Comment:
Additionally, "shortname" is inconsistent with the "short name" stated above; but even more so, AWS doesn't talk about short names for roles. Perhaps we should simply state this as the role name.